src/microbe_stage/PlayerHoverInfo.cs

```diff
@@ -6,13 +6,16 @@
 /// </summary>
 public class PlayerHoverInfo : Node
 {
+    private readonly Dictionary<Compound, float> currentHoveredCompounds = new Dictionary<Compound, float>();
```

Comment:
Could you add a comment on this why this exists and how this differs from HoveredCompounds?